src/box/lua/stat.c

```diff
@@ -77,6 +77,14 @@ inject_iproto_stats(struct lua_State *L, struct iproto_stats *stats)
 			    stats->requests_in_progress);
 	inject_current_stat(L, "REQUESTS_IN_STREAM_QUEUE",
 			    stats->requests_in_stream_queue);
+	lua_pushstring(L, "STREAM_QUEUE_MAX");
+	lua_newtable(L);
+	lua_pushstring(L, "total");
+	lua_pushnumber(L, stats->historical_stream_queue_length_max);
+	lua_rawset(L, -3);
+	lua_rawset(L, -3);
+	inject_current_stat(L, "STREAM_QUEUE_MAX",
+			    stats->stream_queue_length_max);
```

Comment:
Where's "current"?

Reply:
You don't like name "current" for stream_queue_length_max? I talk with @Mons about it, he tell that "total" and "current" is a good name for this metric

Reply:
The name is okay. I didn't understand how you push the "current" string literal to the Lua stack. Now I see that you do this in inject_current_stat. So everything should be fine.